tools/io.go

```diff
@@ -16,9 +17,20 @@ func OpenFileOrFail(filePath string) *os.File {
 	return file
 }
 
-func GetExecutablePath() string {
-	_, b, _, _ := runtime.Caller(0)
```

Comment:
@mfbonfigli , I searched for the path related use method of Go and found the following method
```
b, _ := os.Getwd()
```
The *gocesiumtiler.exe* and assets directory are placed together, so I understand that there is no need to make a definition of 'GOCESIUMTILER_WORKDIR', How about you ?

Reply:
Of course you may have your own considerations, above are my personal thoughts.Thank you :)